modules/sdpolicy-am-aws-compute-types-non-ccs.adoc

```diff
@@ -0,0 +1,44 @@
+
+// Module included in the following assemblies:
+//
+// * assemblies/osd-service-definition.adoc
+:_content-type: CONCEPT
+[id="aws-compute-types-non-ccs_{context}"]
+= AWS compute types for standard clusters
+
+{product-title} offers the following worker node types and sizes on AWS:
+
+.General purpose
```

Comment:
Perhaps you can incorporate this PR here: https://github.com/openshift/openshift-docs/pull/47807